mobile-widgets/qml/DiveDetailsEdit.qml

```diff
@@ -96,6 +96,9 @@ Item {
 			TextField {
 				id: txtDate;
 				Layout.fillWidth: true
+				onEditingFinished: {
+					focus = false
```

Comment:
No those are not related, textfields are not autocompleting.
The autocompleting fields are suit, buddy, DM, and cylinder.